Makefile

```diff
@@ -1,21 +1,20 @@
-ASSETS := $(shell yq e '.assets.[].src' manifest.yaml)
-ASSET_PATHS := $(addprefix assets/,$(ASSETS))
 THUNDERHUB_SRC := $(shell find ./thunderhub)
 
-
 .DELETE_ON_ERROR:
 
-all: thunderhub.s9pk
+all: verify
+
+verify: thunderhub.s9pk
+	embassy-sdk verify thunderhub.s9pk
 
 install: thunderhub.s9pk
-	appmgr install thunderhub.s9pk
+	embassy-cli package install thunderhub.s9pk
 
-thunderhub.s9pk: manifest.yaml config_spec.yaml config_rules.yaml image.tar instructions.md
-	appmgr -vv pack $(shell pwd) -o thunderhub.s9pk
-	appmgr -vv verify thunderhub.s9pk
+thunderhub.s9pk: manifest.yaml assets/compat/config_spec.yaml assets/compat/config_rules.yaml image.tar instructions.md
+	embassy-sdk pack
 
 Dockerfile: $(THUNDERHUB_SRC)
-	cp thunderhub/arm32v7.Dockerfile Dockerfile
+	cp thunderhub/arm64v8.Dockerfile Dockerfile
```

Comment:
figure out how to update the --tag flag to name the image to the format we need: `--tag start9/thunderhub/main:${VERSION}`